tools/lint-hotkeys

```diff
@@ -85,6 +92,27 @@ def generate_hotkeys_file() -> None:
     print(f"Hot Keys list saved in {OUTPUT_FILE}")
 
 
+def format_key(key: str) -> str:
+    """
+    Format each key of a hotkey combination for display
+    """
+    if len(key) == 1:
+        if key.isupper():
+            return f"<kbd>Shift</kbd> + <kbd>{key}</kbd>"
+        return f"<kbd>{key.capitalize()}</kbd>"
+    return f"<kbd>{key}</kbd>"
+
+
+def format_key_combination(key_combination: str) -> str:
+    """
+    Format a hotkey combination for display
+    """
+    for symbol in ARROW_SYMBOLS:
+        if symbol in key_combination:
+            return f"<kbd>{key_combination}</kbd>"
+    return " + ".join([format_key(key) for key in key_combination.split()])
```

Comment:
Just added for an alternative option.
Also, there's the limitation that `format_key_combination()` does not work when the arrow key is used with modifier keys.